design_drafts/components_architecture_and_urdf_examples.md

```diff
@@ -71,31 +71,32 @@ Note:
     </hardware>
     <joint name="joint1">
       <classType>ros2_control_components/MultiInterfaceJoint</classType>
-      <commandInterfaceType>position</commandInterfaceType>
-      <commandInterfaceType>velocity</commandInterfaceType>
-      <commandInterfaceType>effort</commandInterfaceType>
+      <commandInterfaceType name="position">
+        <param name="min_value">-1</param>
+        <param name="max_value">1</param>
+      </commandInterfaceType>
+      <commandInterfaceType name="velocity">
+        <param name="min_value">-1</param>
+        <param name="max_value">1</param>
+      </commandInterfaceType>
+      <commandInterfaceType name="effort">
+        <param name="min_value">-0.5</param>
+        <param name="max_value">0.5</param>
+      </commandInterfaceType>
       <stateInterfaceType>position</stateInterfaceType>
```

Comment:
I think we should allow (and show) that min/max values are optional on state interfaces. This is especially useful for sensors to check received data. Also, this could be application related and do not have to correspond to the min/max values of a sensor.

Reply:
we should do this for sensors first